src/audio/host-zephyr.c

```diff
@@ -1048,7 +1051,8 @@ static int host_reset(struct comp_dev *dev)
 	comp_dbg(dev, "host_reset()");
 
 	if (hd->chan) {
-		dma_stop(hd->chan->dma->z_dev, hd->chan->index);
+		if (dev->state == COMP_STATE_ACTIVE)
```

Comment:
is this check necessary? Is there a possibility of a reset while the comp is still active?

Reply:
I'm not sure. If not, the call to dma_stop should be removed from here because dma has been stopped during PPL pause.

Reply:
I think we should leave the check here. I think for IPC4, it is not possible but maybe we need it for IPC3.